presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -296,6 +296,10 @@ Option                          Description
 
 Troubleshooting
 ---------------
+Enable Insecure access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you enable HTTPS for Presto coordinator, the default behaviour will prevent the access to web interface over HTTP with error ``Web Interface is disabled``. Although not recommended, this can be reenabled by setting
```

Comment:
I don't think you need `the` before `access`, but I do think you need it before `error`:
```
the default behaviour will prevent access to web interface over HTTP with the error ``Web Interface is disabled``
```
The most common problem is when people have an HTTPS proxy in front of the cluster and have not enabled the processing of the forwarded headers.

Reply:
Can this be moved to a troubleshooting section at the bottom of the web interface page instead? Its is not really related to LDAP only

Reply:
Beyond that I suggest to rephrase a bit avoiding future tense and reordering the sentence.

Presto disables HTTP access to the web interface when HTTPS is enabled. Although not recommended, you can enable it by setting ...

Reply:
@mosabua @dain Thanks for the suggestions. I will rectify soon and will let you know for the review.